authenticator/idtoken.go

```diff
@@ -59,8 +59,8 @@ func (s *idTokenAuthenticator) Authenticate(w http.ResponseWriter, r *http.Reque
 	userID, err := claims.UserID()
 	if err != nil {
 		// No USERID_CLAIM, pass this authenticator
```

Comment:
is this comment still applicable?

Also, I think we should add a better comment here explicitly saying the token at this point is valid (it is verified on line 47). In this flow we do not expect a UserID, thus we can log and continue.

Reply:
Updated this.